tests/e2e/tools/openMeteo.test.ts

```diff
@@ -42,15 +42,57 @@ describe("OpenMeteo", () => {
 
     expect(response.isEmpty()).toBe(false);
     expect(response.result).toMatchObject({
-      latitude: expect.any(Number),
-      longitude: expect.any(Number),
-      generationtime_ms: expect.any(Number),
-      utc_offset_seconds: 0,
       timezone: "UTC",
-      timezone_abbreviation: "UTC",
+      current: expect.any(Object),
+      current_units: expect.any(Object),
+      hourly: expect.any(Object),
+      hourly_units: expect.any(Object),
+      daily: expect.any(Object),
+      daily_units: expect.any(Object),
     });
   });
 
+  it("Filters", async () => {
+    instance = new OpenMeteoTool({
+      responseFilter: {
+        omittedResponseKeys: [
+          "latitude",
+          "longitude",
+          "timezone_abbreviation",
+          "elevation",
+          "utc_offset_seconds",
+          "generationtime_ms",
+          "timezone",
+          "hourly",
+          "hourly_units",
```

Comment:
You originally had these values as defaults, but you deleted them. I might keep them as default.

Reply:
I added a default response filter with these keys excluded.